pyproject.toml

```
@@ -1,10 +1,89 @@
 [build-system]
 requires = [
-  "setuptools",
-  "setuptools_scm",
-  "wheel",
+  "setuptools>=62.1",
+  "setuptools_scm[toml]>=6.2",
+  "wheel",]
```

There should be a line break here:

```suggestion
  "wheel",
]
```

I think the whitespace stripping here: 

https://github.com/sunpy/package-template/blob/6d6729b22066ef890e70c37438da70cac33e03b4/%7B%7B%20cookiecutter.package_name%20%7D%7D/pyproject.toml#L10

is back to front, it should be `{%- endif %}`